core/src/main/java/com/klaytn/caver/methods/response/KlayBlock.java

```diff
@@ -137,7 +142,7 @@ public Block getBlock() {
         /**
          * Array of transaction objects, or 32-byte transaction hashes depending on the last given parameter
          */
-        private List<KlayTransaction.Transaction> transactions;
+        private List transactions;
```

Comment:
In this situation, transactions accepts any types. Did you consider using Generic?

Reply:
This was just a question. If you think Generic version looks ugly and uncomfortable, you could change previous version which does not have Generic